_es/tour/basics.md

```diff
@@ -6,4 +6,319 @@ discourse: false
 
 partof: scala-tour
 language: es
+
+num: 2
+next-page: unified-types
+previous-page: tour-of-scala
 ---
+
+En esta página, cubriremos lo básico de Scala.
+
+## Probando Scala en el navegador
+
+Puedes ejecutar Scala en tu navegador con ScalaFiddle.
+
+1. Ve a [https://scalafiddle.io](https://scalafiddle.io).
+2. Escribe `println("Hello, world!")` en el panel a la izquierda.
+3. Presiona el botón "Run". En el panel de la derecha aparecerá el resultado.
+
+Así, de manera fácil y sin preparación, puedes probar fragmentos de código Scala.
+
+Muchos ejemplos de código en esta documentación están integrados con ScalaFiddle, y así puedes probarlos directamente solo con pulsar el botón "Run".
+
+## Expresiones
+
+Las expresiones son sentencias computables.
+
+```tut
+1 + 1
+```
+
+Se puede ver el resultado de evaluar expresiones usando `println`.
+
+{% scalafiddle %}
+```tut
+println(1) // 1
+println(1 + 1) // 2
+println("Hello!") // Hello!
+println("Hello," + " world!") // Hello, world!
+```
+{% endscalafiddle %}
+
+## Valores
+
+Se puede dar un nombre al resultado de una expresión usando la palabra reservada `val`.
+
+```tut
+val x = 1 + 1
+println(x) // 2
+```
+
+Los resultados con nombre, como `x` en el ejemplo, son llamados valores. Referenciar un valor no lo vuelve a computar.
+
+Los valores no pueden ser reasignados.
+
+```tut:fail
+x = 3 // This does not compile.
+```
+
+Los tipos de los valores pueden ser inferidos, pero también se pueden anotar explicitamente de la siguiente forma:
+
+```tut
+val x: Int = 1 + 1
+```
+
+Noten como la declaración de tipo `Int` va después del identificador `x`. También necesitas un `:`.
+
+## Variables
+
+Una variable es como un valor, excepto que a una variable se le puede re-asignar un valor después de declararla. Una variable se declara con la palabra reservada `var`.
+
+```tut
+var x = 1 + 1
+x = 3 // This compiles because "x" is declared with the "var" keyword.
```

Comment:
This comment could be translated too, but I'm beginning to think that translating no comment could also be OK too. I'll leave it up to you to pick a consistent approach (translate comments VS no translate them).

Reply:
yeah, I went back to NOT translate code (& code comments)